utils/suggestions.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {AsciiToUtf8Characters} from './constants';
+
+export const asciiToUtf8Variants = (term: string) => {
+    let variants = [term];
+    term.split('').forEach((t: string, i: number) => {
+        if ((AsciiToUtf8Characters as {[key: string]: string[]})[t] && i < 5) {
+            (AsciiToUtf8Characters as {[key: string]: string[]})[t].forEach((utf8char: string) => {
+                const substr = asciiToUtf8Variants(term.replace(t, utf8char));
```

Comment:
This isn't relevant if we change how we do this, but I'm not sure about using recursion here since:
1. This ends up looking at characters we've already looked at before.
2. I think this misses some cases where a non-accented character appears before an accented one. For example, if you give this `DY` as input, I think it'd return `['DY', 'ÐY', 'ÐÝ']` without `DÝ`.